src/commands/subcommands/tasks.ts

```diff
@@ -0,0 +1,104 @@
+import { HttpStatusCode, IModify } from '@rocket.chat/apps-engine/definition/accessors';
+import { SlashCommandContext } from '@rocket.chat/apps-engine/definition/slashcommands';
+import { LayoutBlock } from '@rocket.chat/ui-kit';
+
+import { TodoistApp } from '../../../TodoistApp';
+import { MiscEnum } from '../../enums/Misc';
+import {
+  getActionsBlock,
+  getButton,
+  getContextBlock,
+  getSectionBlock,
+} from '../../helpers/blockBuilder';
+import { getTasksUrl } from '../../helpers/const';
+import { ITask } from '../../interfaces/tasks';
+
+export async function tasks(
```

Comment:
I think we should have a service to talk with the network, refer back to what I pointed out in labels